website/sitePlugin.js

```diff
@@ -0,0 +1,16 @@
+const webpack = require("webpack");
+
+module.exports.default = function sitePlugin() {
+  return {
+    name: "sitePlugin",
+    configureWebpack(config) {
+      return {
+        plugins: [
+          new webpack.EnvironmentPlugin({
+            FB_INTERNAL: process.env.FB_INTERNAL ?? false,
```

Comment:
temporarily required, will likely be fixed in the fb custom preset by @jknoxville